app/errors/insufficient_gas_price_test.go

```diff
@@ -56,7 +57,7 @@ func TestInsufficientMinGasPriceIntegration(t *testing.T) {
 	require.NoError(t, err)
 
 	_, err = anteHandler(ctx, sdkTx, false)
-	require.True(t, apperr.IsInsufficientMinGasPrice(err))
+	require.True(t, apperr.IsInsufficientFee(err))
```

Comment:
[nit] this test could be renamed `TestInsufficientMinGasPriceIntegration` -> `TestInsufficientFee`